app/controllers/faqs_controller.rb

```diff
@@ -0,0 +1,68 @@
+class FaqsController < ApplicationController
+  before_action :set_faq, only: %i[ show edit update destroy ]
+
```

Comment:
Let allow non authenticated users to see FAQ: add `skip_before_action :authenticate_user!` to the controller for the index method only